python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -116,6 +117,17 @@ def in_progress() -> "InProgressSchedulingCondition":
 
         return InProgressSchedulingCondition()
 
+    @staticmethod
+    def updated_since_cron(
+        cron_schedule: str, cron_timezone: str = "UTC"
+    ) -> "UpdatedSinceCronCondition":
+        """Returns a SchedulingCondition that is true for an asset partition if it has been updated
```

Comment:
Example with concrete times very helpful in the doc block. I find with rules like this super concretes examples much easier to understand than the abstract description.